Jenkinsfile

```diff
@@ -622,6 +663,42 @@ try {
           }
         }
       }
+    },
+    'Python 2: MKLDNN-GPU Win':{
+      node('mxnetwindows-gpu') {
+        timeout(time: max_time, unit: 'MINUTES') {
+          ws('workspace/ut-python-gpu') {
+          init_git_win()
+          unstash 'vc14_gpu_mkldnn'
+          bat '''rmdir /s/q pkg_vc14_gpu_mkldnn
+            7z x -y vc14_gpu_mkldnn.7z'''
+          bat """xcopy C:\\mxnet\\data data /E /I /Y
+            xcopy C:\\mxnet\\model model /E /I /Y
+            call activate py2
+            set PYTHONPATH=${env.WORKSPACE}\\pkg_vc14_gpu_mkldnn\\python
+            del /S /Q ${env.WORKSPACE}\\pkg_vc14_gpu_mkldnn\\python\\*.pyc
+            C:\\mxnet\\test_gpu.bat"""
+          }
+        }
+      }
+    },
+    'Python 3: MKLDNN-GPU Win':{
```

Comment:
i'm always wondering if it's necessary to test both Python 2 and 3 for all different cases? @marcoabreu

Reply:
I think only keep one python 2 test is ok.

Reply:
Yes, please only keep one test. Since you're testing the CPP backend, which is independent of Python, there's no need to test both versions.

Reply:
ok